frontend/src/scenes/session-recordings/player/playlist-popover/PlaylistPopover.tsx

```diff
@@ -90,39 +90,44 @@ export function PlaylistPopoverButton({
 
                         {allPlaylists.length ? (
                             <div className="max-h-60 overflow-auto">
-                                {allPlaylists?.map(({ selected, playlist }) => (
-                                    <div key={playlist.short_id} className="flex items-center gap-1">
-                                        <LemonButton
-                                            className="flex-1"
-                                            icon={
-                                                currentPlaylistsLoading &&
-                                                modifyingPlaylist?.short_id === playlist.short_id ? (
-                                                    <Spinner className="text-sm" />
-                                                ) : (
-                                                    <LemonCheckbox className="pointer-events-none" checked={selected} />
-                                                )
-                                            }
-                                            onClick={() => {
-                                                if (
-                                                    setPinnedInCurrentPlaylist &&
-                                                    playlist.short_id === currentPlaylistId
-                                                ) {
-                                                    return setPinnedInCurrentPlaylist(!selected)
+                                {allPlaylists
+                                    ?.filter((p) => p.playlist.short_id !== 'history')
```

Comment:
remove history here, since you can't add things to the history collection except by watching them
but you can find something in your history and then add it to a different collection